paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -69,6 +70,18 @@ public abstract class PrepareResourcesTask : DefaultTask() {
   @get:OutputFile
   public abstract val paparazziResources: RegularFileProperty
 
+  // Used to require AAR base resource directories are inputs
+  @get:Input
+  public val aarExplodedResourcesBaseDirs: Provider<List<String>> = project.provider {
+    aarExplodedDirs.relativize(gradleUserHomeDirectory)
+  }
+
+  // Used to require AAR base asset directories are inputs
+  @get:Input
+  public val aarExplodedAssetsBaseDirs: Provider<List<String>> = project.provider {
+    aarAssetDirs.relativize(gradleUserHomeDirectory)
+  }
```

Comment:
Nice!

It looks like the original `aarExplodedDirs` and `aarAssetDirs` property types should change instead of adding more properties, so how about removing these changes and instead modifying the originals like so:

```
  @get:Input
  public abstract val aarExplodedDirs: ListProperty<String>
```

then move that logic into the Plugin, there's a bit of prior art there!